### DISCUSSION TITLE
Privacy

### DISCUSSION THREAD

**Discussion by Charles-Pt4** [exadel-inc/CompreFace#1046] [General]
Hello, we would like to use the compreface to identify the gender and count people. I would like to know if anyone already use this system in public place and how it works with privacy. When we use the api the send image to analyse, is the image are stored on the server ? Thank you,
Charles

**Comment by pospielov** [comment]
We don't know if somebody uses CompreFace like this. And we don't have such a business case in our own practice yet.
We save images only in the recognition service. And it may be disabled `save_images_to_db` setting in `.env` file to false.
I think "How it works with privacy" depends on the country. I think it's important not to make any decisions for individual persons based on recognized gender, as the model is not 100% accurate, but aggregation should be fine, but I'm not a legal expert. You can try to follow Microsoft ["Responsible use of AI"](https://learn.microsoft.com/en-us/legal/cognitive-services/computer-vision/transparency-note-spatial-analysis) rules. It doesn't contain gender detection, but I think any automatic video processing should be similar from the law's perspective. 
I also would recommend using [custom build](https://github.com/exadel-inc/CompreFace/blob/master/docs/Custom-builds.md), as its age plugin is more accurate.

**Reply by Charles-Pt4** [reply]
Thank you for the feedback. I will check the "Responsible use of AI" you shared.
